.github/workflows/log_binary_size.yml

```diff
@@ -32,18 +46,10 @@ jobs:
       run: |
         git config --local user.name 'TFLM-bot'
         git config --local user.email 'tflm-github-bot@google.com'
-        git commit -am "Automated size log update"
-    - name: Create Logs PR Request
-      id: create-pr
-      uses: peter-evans/create-pull-request@052fc72b4198ba9fbc81b818c6e1859f747d49a8
-      with:
-         branch: nightly_binary_size_profiling
-         delete-branch: true
-         token: ${{ secrets.TFLM_BOT_REPO_TOKEN }}
-         title: Automated binary size log update
-         commit-message: Automated binary size log update
-         committer: TFLM-bot <tflm-github-bot@google.com>
-         author: TFLM-bot <tflm-github-bot@google.com>
-         body: "BUG=automated binary size log update"
-         labels: ci:run
-         reviewers: advaitjain
+        git commit -a -m "Automated binary size log update"
```

Comment:
after commit, there is no push. but then there is a git checkout $LOG_BRANCH (even though I think we are are already on this branch).  Could you please help elaborate on this sequence?

Reply:
Also, is this just committing the updated profile file? 

I am guessing the code in $LOG_BRANCH will stay forever stale, but what is in data/continous_builds/size_profiling will keep updated based on latest code.  Is this a necessary trade-off that we have to live by or could other directories/files in LOG_BRANCH also get updated (without impacting the continuous update of  data/continous_builds/size_profiling of couse)?

Reply:
The lack of a push is intentional, we are making no changes in main. However, to checkout the modified files from main, they must have been committed. It's just how git works. Your changes can't be accessed from another branch until the files are committed. 
I see how this is confusing. Give me a few minutes and I will do a little explanation of what is going on here with steps, cause yeah, if you don't get what is happening this looks off and isn't intuitive to everyday use at all.

Reply:
As for updating, we can definitely update anything we want in the $LOG_BRANCH. Only the files being affected by the test being run here will be pulled over. This is specifically as generic as possible so we can use this with other test suites that accumulate housekeeping files.